datahub-web-react/src/app/glossary/EmptyGlossarySection.tsx

```diff
@@ -31,6 +32,9 @@ function EmptyGlossarySection(props: Props) {
     const [isCreateTermModalVisible, setIsCreateTermModalVisible] = useState(false);
     const [isCreateNodeModalVisible, setIsCreateNodeModalVisible] = useState(false);
 
+    const { entityData } = useEntityData();
+    const canCreateGlossaryEntity = !!entityData?.privileges?.canManageChildren;
```

Comment:
this `EmptyGlossarySection` is also used by the BusinessGlossaryPage component, so `entityData` could very well be empty when we're not on an entity page. Can you also add a check to see if the user has the manageGlossaries privilege before disabling?